guides/ai/mcp.mdx

```diff
@@ -0,0 +1,210 @@
+
+1. **Open Claude Desktop** and start a new conversation.
+
+2. **Connect to your Meilisearch instance** by asking Claude to update the connection settings:
+
+   ```
+   Please connect to my Meilisearch instance at https://ms-xxx.meilisearch.io using the API key XXXXXX
+   ```
+
+   Claude will use the MCP server's `update-connection-settings` tool to establish a connection to your Meilisearch instance.
+
+3. **Verify the connection** by asking:
+
+   ```
+   Can you check the connection to my Meilisearch instance and tell me what version it's running?
+   ```
+
+   Claude will use the `get-version` and `health-check` tools to verify the connection and provide information about your instance.
+
+## Example Queries and Tasks
+
+Here are some examples of what you can ask Claude to do with your Meilisearch instance:
+
+### Managing Indexes
+
+**Create an index optimized for e-commerce:**
+```
+Create a new index called "products" for our e-commerce site with the primary key "product_id"
+```
+
+**List all indexes:**
+```
+What indexes do I have in my Meilisearch instance?
+```
+
+**Get index metrics:**
+```
+How many documents are in my "products" index and what's its size?
+```
+
+### Document Operations
+
+**Add documents to an index:**
+```
+Add these products to my "products" index:
+[
+  {"product_id": 1, "name": "Ergonomic Chair", "description": "Comfortable office chair", "price": 299.99, "category": "Furniture"},
+  {"product_id": 2, "name": "Standing Desk", "description": "Adjustable height desk", "price": 499.99, "category": "Furniture"}
+]
+```
+You can also add fake data for testing purposes
+```
+Add 10 documents in the index 'movies' with a title and description of your choice
+```
+
+**Retrieve documents:**
+```
+Show me the first 5 products in my "products" index
+```
+
+### Search Configuration
+
+**Update ranking rules:**
+```
+Update the ranking rules for the "products" index to prioritize word matches and handle typos, but make exact matches more important than proximity
+```
+
+**Configure search settings:**
+```
+Configure my "products" index to make the "name" and "description" fields searchable, but only "name", "price", and "category" should be displayed in results
+```
+
+### Performing Searches
+
+**Basic search:**
+```
+Search the "products" index for "desk" and return the top 3 results
+```
+
+**Advanced search with filters:**
+```
+Search the "products" index for "chair" where the price is less than 200 and the category is "Furniture". Sort results by price in ascending order.
+```
+
+### Monitoring and Maintenance
+
+**Check system health:**
+```
+Is my Meilisearch instance healthy? What are the current stats?
+```
+
+**Manage tasks:**
+```
+Show me the status of recent indexing tasks
+```
+
+**API key management:**
+```
+Create a new API key with read-only access to the "products" index that expires in 30 days
+```
+
```

Comment:
I'm not sure, I feel these don't fit within the main task of the guide and am inclined to say we can remove them. A guide that is short and to the point is more likely to be read than a long guide with a lot of content and tries to cover too many topics.

````suggestion
````

Reply:
I agree 👍🏽